braph2genesis/measures/_OutGlobalEfficiency.gen.m

```diff
@@ -0,0 +1,286 @@
+%% ¡header!
+OutGlobalEfficiency < Distance (m, out-global efficiency) is the graph out-global efficiency.
+
+%%% ¡description!
+The out-global efficiency is the average inverse shortest in-path length within each layer. 
+It is inversely related to the characteristic in-path length.
+
+%% ¡props_update!
+
+%%% ¡prop!
+NAME (constant, string) is the name of the OutGlobalEfficiency.
+%%%% ¡default!
+'OutGlobalEfficiency'
+
+%%% ¡prop!
+DESCRIPTION (constant, string) is the description of the OutGlobalEfficiency.
+%%%% ¡default!
+'The out-global efficiency is the average inverse shortest in-path length within each layer. It is inversely related to the characteristic in-path length.'
+
+%%% ¡prop!
+TEMPLATE (parameter, item) is the template of the OutGlobalEfficiency.
+%%%% ¡settings!
+'OutGlobalEfficiency'
+
+%%% ¡prop!
+ID (data, string) is a few-letter code of the OutGlobalEfficiency.
+%%%% ¡default!
+'OutGlobalEfficiency ID'
+
+%%% ¡prop!
+LABEL (metadata, string) is an extended label of the OutGlobalEfficiency.
+%%%% ¡default!
+'OutGlobalEfficiency label'
+
+%%% ¡prop!
+NOTES (metadata, string) are some specific notes about the OutGlobalEfficiency.
+%%%% ¡default!
+'OutGlobalEfficiency notes'
+
+%%% ¡prop!
+SHAPE (constant, scalar) is the measure shape __Measure.NODAL__.
+%%%% ¡default!
+Measure.NODAL
+
+%%% ¡prop!
+SCOPE (constant, scalar) is the measure scope __Measure.UNILAYER__.
+%%%% ¡default!
+Measure.UNILAYER
+
+%%% ¡prop!
+PARAMETRICITY (constant, scalar) is the parametricity of the measure __Measure.NONPARAMETRIC__.
+%%%% ¡default!
+Measure.NONPARAMETRIC
+
+%%% ¡prop!
+COMPATIBLE_GRAPHS (constant, classlist) is the list of compatible graphs.
+%%%% ¡default!
+{'GraphWD' 'GraphBD' 'MultiplexWD' 'MultiplexBD' 'OrdMxBD' 'OrdMxWD'} ;%TBE % % % add multilayerWD multilayerBD ordmlwd ordmlbd tests
+
+%%% ¡prop!
+M (result, cell) is the out-global efficiency.
+%%%% ¡calculate!
+g = m.get('G');  % graph from measure class
+A = g.get('A'); % cell with adjacency matrix (for graph) or 2D-cell array (for multigraph, multiplex, etc.)
+L = g.get('LAYERNUMBER');
+N = g.get('NODENUMBER');
+
+distance = calculateValue@Distance(m, prop);
+
+out_global_efficiency = cell(L, 1);
+parfor li = 1:1:L
+    inverse_distance = distance{li}.^-1;  % inverse distance
+    inverse_distance(1:N(li)+1:end) = 0;
+    global_efficiency_layer = (sum(inverse_distance, 2) / (N(li)-1));
+    out_global_efficiency(li) = {global_efficiency_layer};
+end
+value = out_global_efficiency;
+
+%% ¡tests!
+
+%%% ¡excluded_props!
+[OutGlobalEfficiency.PFM]
+
+%%% ¡test!
+%%%% ¡name!
+GraphBD
+%%%% ¡code!
+B = [
+    0   .1  0   0   0
+    .2  0   0   0   0
+    0   0   0   .2  0
+    0   0   .1  0   0
+    0   0   0   0   0
+    ];
+
+known_out_global_efficiency = {[1/4 1/4 1/4 1/4 0]};
```

Comment:
Here I think it need to be the transpose (so we have to add ' to the end. Like this: known_out_global_efficiency = {[1/4 1/4 1/4 1/4 0]'}